pkgs/tools/misc/nagstamon/default.nix

```
@@ -2,26 +2,23 @@
 
 pythonPackages.buildPythonApplication rec {
   pname = "nagstamon";
-  version = "3.2.1";
+  version = "3.14.0";
 
   src = fetchurl {
-    url = "https://nagstamon.ifw-dresden.de/files/stable/Nagstamon-${version}.tar.gz";
-    sha256 = "1048x55g3nlyyggn6a36xmj24w4hv08llg58f4hzc0fwg074cd58";
+    url = "https://github.com/HenriWahl/Nagstamon/archive/refs/tags/v${version}.tar.gz";
+    sha256 = "sha256-9RxQ/rfvoyjSUsY4tmAkBdVQqZYi3X6PBzQYFIeenzA=";
   };
 
   # Test assumes darwin
   doCheck = false;
 
-  propagatedBuildInputs = with pythonPackages; [ configparser pyqt5 psutil requests
-     beautifulsoup4 keyring requests-kerberos kerberos lxml ];
+  propagatedBuildInputs = with pythonPackages; [ configparser pyqt6 psutil requests
+     beautifulsoup4 keyring requests-kerberos lxml dbus-python dateutil pysocks setuptools ];
```

ofborg complains that there is no such package named `python3Packages.dateutil`. Maybe this was what you meant?

```suggestion
     beautifulsoup4 keyring requests-kerberos lxml dbus-python python-dateutil pysocks setuptools ];
```

---

Im refer to [python311Packages.dateutils](https://search.nixos.org/packages?channel=24.05&from=0&size=50&sort=relevance&type=packages&query=dateutil)

---

Then you're missing an `s` at the end

---

I'm a bit confused. Currently, you have changed the package name from `dateutil` to `python-dateutils`, despite saying you were aiming to add `dateutils`. It's not immediately clear to me which one it is by reading upstream.

Could you verify that it is correct?

---

I yes i confirm is `python-dateutil` not dateutil or dateutils